test/text/test_text_mode.py

```diff
@@ -0,0 +1,32 @@
+from pathlib import Path
+
+from fpdf import FPDF, TextMode
+from test.conftest import assert_pdf_equal
+
+
+HERE = Path(__file__).resolve().parent
+
+
+def test_text_mode_stroke(tmp_path):
+    pdf = FPDF(orientation="landscape")
+    pdf.add_page()
+    pdf.set_font("Helvetica", size=80)
+    with pdf.local_context(text_mode=TextMode.STROKE, line_width=2):
```

Comment:
Seems to work well enough.

Full testing and comprehensive documentation are obviously still missing, but I'm sure you're aware of that. :wink:

Reply:
Actually I'd be curious to know what you mean by "full testing" (what cases do you think should be covered?) and comprehensive documentation (what should it contains? should we repeat part of the PDF spec?)

Reply:
I'd expect to see all the options demonstrated in the tests, ideally with various permutations of parameters (eg. line_width). Essentially to visually verify that we're actually triggering the correct behaviour in the PDF reader in all cases.

The documentation should probably also give a short description of all the options, including which other parameters each of them can interact with. Eg.: Are there any other interactions than with line_width? Does the stroke outline use draw_color or text_color? The paragraph and image you added to TextStyling.md are a good start to that, but only show two of the many options.

I wouldn't expect our users to study the PDF specs themselves. We don't have to literally "repeat" them, since we operate on a higher level of abstraction, but I think there should be a general explenation of all the effects that we offer access to with the library.

Reply:
I added extra explanations & examples